apps/tailwind-components/components/Input.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <InputString
```

Comment:
I had problems with Lazy during the tests, now try without Lazy to see if that helps